spring-rabbit/src/main/java/org/springframework/amqp/rabbit/support/micrometer/RabbitListenerObservation.java

```diff
@@ -63,14 +60,36 @@ public enum ListenerLowCardinalityTags implements KeyName {
 		 * Listener id.
 		 */
 		LISTENER_ID {
-
 			@Override
 			public String asString() {
 				return "spring.rabbit.listener.id";
 			}
 
-		}
+		},
+
+		/**
+		 * The exchange the listener is plugged to (empty if default exchange).
```

Comment:
The listener deals with queues, not exchanges.
So, I believe here this has to be set into `context.getCarrier().getMessageProperties().getConsumerQueue()`.

Reply:
done